utoipa-swagger-ui/README.md

```diff
@@ -31,25 +31,43 @@ more details at [serve](https://docs.rs/utoipa-swagger-ui/latest/utoipa_swagger_
 * **debug-embed** Enables `debug-embed` feature on `rust_embed` crate to allow embedding files in debug
   builds as well.
 
-# Install
+## Install
 
 Use only the raw types without any boilerplate implementation.
+
 ```toml
 [dependencies]
 utoipa-swagger-ui = "6"
 ```
 
 Enable actix-web framework with Swagger UI you could define the dependency as follows.
+
 ```toml
 [dependencies]
 utoipa-swagger-ui = { version = "6", features = ["actix-web"] }
 ```
 
 **Note!** Also remember that you already have defined `utoipa` dependency in your `Cargo.toml`
 
-# Examples
+## Config
+
+The following configuration env variables are available at build time:
+
+* `SWAGGER_UI_DOWNLOAD_URL`:
+
+  * the url from where to download the swagger-ui zip file
+  * default value: <https://github.com/swagger-api/swagger-ui/archive/refs/tags/v5.3.1.zip>
```

Comment:
```suggestion
  * default value: <https://github.com/swagger-api/swagger-ui/archive/refs/tags/v5.11.0.zip>
```

Reply:
This one probably should be changed as well

Reply:
Ah yes indeed! I see you've done it.

Reply:
Yeah, I thought just directly commit it to get this thing merged faster :smile: As change was anyway just a version number on the README.md :slightly_smiling_face: